src/models/search.js

```diff
@@ -55,18 +55,37 @@ export default {
       searchResults.resultCount = response.hits.total;
```

Comment:
It seems `resultCount` is being assigned an object opposed to an integer specified from the `total` field. This is causing `[Object] [object]` to be rendered in the Search page component once a search query has been made.

Reply:
Thanks. Somehow I missed that, both visually and comparing my Postman ES7 output... hits.total is an object `{value: 1, relation: 'eq'}` ... I want `response.hits.total.value` here.

Reply:
![image](https://user-images.githubusercontent.com/61761529/104379483-816d2080-54f7-11eb-9fe4-67513224d56b.png)